content/docs/handling-events.md

```diff
@@ -1,43 +1,43 @@
 ---
 id: handling-events
-title: Handling Events
+title: Manejando Eventos
 permalink: docs/handling-events.html
 prev: state-and-lifecycle.html
 next: conditional-rendering.html
 redirect_from:
   - "docs/events-ko-KR.html"
 ---
 
-Handling events with React elements is very similar to handling events on DOM elements. There are some syntactic differences:
+Manejar eventos con elementos de React es muy similar a manejar eventos con elementos del Dom.. Solo hay una diferencias de sintaxis:
 
-* React events are named using camelCase, rather than lowercase.
-* With JSX you pass a function as the event handler, rather than a string.
+* Los eventos de React se nombran usando camelCase, en vez de minúsculas.
+* En JSX se pasa la función como el manejador del evento, en vez de una cadena de texto.
```

Comment:
Could be:
```suggestion
* Con JSX pasas una función como el manejador del evento, en vez de un string.
```
Here are some words that you should consider to not translate, as string: https://github.com/reactjs/es.reactjs.org/blob/master/TRANSLATION.md